website/src/pages/brand/index.tsx

```diff
@@ -0,0 +1,881 @@
+          Darken 4
+        </button>
+        <button
+          onClick={() => copyText("#030405")}
+          className={styles.btnTextWhite}
+          style={{ backgroundColor: "rgba(0, 0, 0, 0.96)" }}
+        >
+          Darken 5
+        </button>
+      </div>
+    </section>
+
+    <section
+      className={`${styles.sectionGreen} ${styles.gap40}`}
+      id="typography"
+    >
+      <h1>Typography</h1>
+      <h1 className={styles.fw_300}>Inter</h1>
+
+      <div className={styles.paragraphColumn}>
+        <p>
+          Inter is a popular open-source typeface designed by Rasmus Andersson.
+          It was created with the purpose of being a legible, versatile, and
+          modern font suitable for a wide range of digital applications. This
+          includes websites, mobile apps, and user interfaces. Inter features a
+          balanced and consistent design, with various weights and styles that
+          you can use for different purposes, from body text to headlines. Its
+          simple and clean lines make it easy to read on screens of all sizes,
+          and its multilingual support makes it accessible to users from all
+          over the world.
+        </p>
+        <p className={styles.pt_3}>
+          We use Inter for everything from the wordmark in our logo to the
+          branding and the interfaces of our products. Inter is part of{" "}
+          <strong>Google Fonts</strong>.
+        </p>
+      </div>
+
+      <h1>
+        ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890 ?!()[]{}
+        &*^%$#@~
+      </h1>
+
+      <h1 className={styles.fw_300}>
+        The quick brown fox jumps over the lazy dog
+      </h1>
+
+      <h3 className={styles.fw_300}>
+        Le vif renard brun saute par-dessus le chien paresseux
+      </h3>
+    </section>
+
+    <section
+      className={`${styles.sectionPink} ${styles.gap40}`}
+      id="toneofvoice"
+    >
+      <h1>Tone of Voice</h1>
+
+      <div className={styles.paragraphColumn}>
+        <p>Our tone of voice rests on the following principles:</p>
+
+        <ul>
+          <li>
+            <strong>Be casual but knowledgeable</strong>: Find a balance between
+            down to earth and intelligent content.
+          </li>
+          <li>Avoid buzzwords whenever possible.</li>
+          <li>
+            <strong>Be transparent</strong>: Be more open than our competitors
+            about our platform, traffic, and business model
+          </li>
+          <li>
+            <strong>Be positive</strong>: Talk about the telecom industry's
+            operator incumbents and legacy vendors as heros, not villains or
+            idiots
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section className={`${styles.sectionBlue} ${styles.gap40}`} id="downloads">
+      <h1>The good stuff!</h1>
+
+      <h3 className={`${styles.downloadLink} ${styles.fw_300}`}>
+        <a href="#">
+          Download all WG2 logos
```

Comment:
Can you make the download link https://drive.google.com/drive/folders/1gm13n5gH7qRm2jDe9bSRBydNc9xMGBjv